Some compilation errors fixed...

vogje01 opened the pull request:
Hi,

I fixed some compilation errors. Moved to latest Intellij-idea (2018.3). Changed some xtend files, due to API changes in the Intellij SDK.

Still not working. Need some more time.

Cheers
Jens

Commit:
Fix most of the compile time errors.

Commit:
Fix storage issues.

Commit:
Modified .gitignore.

Commit:
Modified .gitignore.

Commit:
Modified .gitignore.

Commit:
Fix encoding

Commit:
Fix some minor warnings.

tivervac commented:
Jenkins appears to have some issues with the new grgit you set up, is that one of the issues you're aware of?

Commit:
Fix encodings.

Commit:
Changed org.ajoberstar.grgit back to 2.2.0
Fixed some compile errors concerning ParserTest.java
Fixed some deprecations.
Fixed some API changes in intellij 2018.3

vogje01 commented:
No sccess
